### DISCUSSION TITLE
Got Questions or Topics for Starknet? Let Us Know!（收集下大家关于starknet的问题和想听的分享主题）

### DISCUSSION THREAD

**Discussion by qijinzing** [coset-io/zkp-academy#39] [General]
hi 大家，正在准备和starknet沟通分享主题，大家有什么想听的方向吗？关于Fri/Stark，可以在这里留言，我们会组织官方来分享，也争取能给大家解答。

Have questions about Fri/Stark or specific topics you’d like to hear more about? Share them with us! We’re collecting your thoughts and ideas to make our discussions as relevant and engaging as possible.

**Comment by wenjin1997** [comment]
我最近在阅读 [[BGKS20] DEEP-FRI](https://eprint.iacr.org/2019/336) 和 [[BCIKS20] Proximity Gaps](https://eprint.iacr.org/2020/654.pdf) 这两篇论文时，注意到它们在分析 FRI 的 soundness 时提到了一个关于 proximity parameter $\delta$ 能达到 list decoding capacity $1 - \rho$ 的猜想，即 "List-Decoding Conjecture"。

我想请教一下，在实际的实现中，是否会基于这个猜想？如果采用这个猜想，会存在什么风险吗？另外，关于这个猜想，目前理论上有什么最新的进展？

非常感谢您的帮助！

---

I have recently been reading the papers [[BGKS20] DEEP-FRI](https://eprint.iacr.org/2019/336) and [[BCIKS20] Proximity Gaps](https://eprint.iacr.org/2020/654.pdf), and I noticed that they mention a conjecture regarding the proximity parameter $\delta$ reaching the list decoding capacity $1 - \rho$ , referred to as the "List-Decoding Conjecture," in their analysis of the soundness of FRI.

I would like to inquire whether this conjecture is used in practical implementations. If so, what are the potential risks associated with adopting this conjecture? Additionally, what are the latest theoretical advancements concerning this conjecture?

Thank you very much for your assistance!

**Comment by qijinzing** [comment]
微信群接龙：

1. Miles 他们的Prover network 设计
2. Harry L 如何用stark验证bitcoin区块
3. Wangyao stwo prover 设计细节
4. Kero starknet做 zk vm 时为什么选择重新设计一门CASM，而不是选择工业界已有的 RISCV/MIPS 等指令集？从zk vm 的角度，CASM相对于RISCV/MIPS有什么优势？
5. Jade 在论文 [BGKS20] DEEP-FRI 以及 [BCIKS20] Proximity Gaps 中，在分析 FRI 的 soundness 时，都提到了一个关于 proximity parameter \delta 能达到 $1 - \rho$ 的猜想，即 "List-Decoding Conjecture"。请问在实际的实现中会基于这个猜想嘛？如果采用会存在什么风险吗？请问关于这个猜想目前理论上有什么新的进展吗？
6. Alan How to choose ε for FRI
7. 笃行 ：你们未来看好的方向是什么?

**Reply by luckyyang** [reply]
1. Miles: Prover network design.
2. Harry L: How to use STARK to verify Bitcoin blocks.
3. Wangyao: Details of the STWO Prover design.
4. Kero: Why choose to redesign CASM for StarkNet zk VM instead of using existing industrial instruction sets like RISC-V/MIPS? What advantages does CASM have over RISC-V/MIPS from the perspective of zk VM?
5. Jade: In the papers [BGKS20] DEEP-FRI and [BCIKS20] Proximity Gaps, when analyzing the soundness of FRI, a conjecture about the proximity parameter $$\delta$$ reaching $$1 - \rho$$ is mentioned, namely the "List-Decoding Conjecture." In practice, is this conjecture relied upon? If so, what risks might it pose? Are there any recent theoretical advancements regarding this conjecture?
6. Alan: How to choose $$\varepsilon$$ for FRI.
7. Question: What are your future directions?
